pos_sale_configurator_option/models/sale_order.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    def _prepare_pos_option(self, option):
+        if option["notes"]:
+            description = "\n".join([option["description"], option["notes"]])
+        else:
+            description = option["description"]
+        return {
+            "product_id": int(option["product_id"]),
+            "option_unit_qty": int(option["quantity"]),
+            "option_qty_type": "proportional_qty",
+            "name": description,
```

Comment:
why no product_uom , product_uom_qty here ? @sebastienbeau

Reply:
option use the uom of the product.  the product_uom_qty will be updated by the option_unit_qty

Reply:
I had this issues but didn't investigate: on PoS set a quantity > 1 on the parent product and the save the order. In backend you will find wrong quantities.